jkq/ddsim/hybridqasmsimulator.py

```diff
@@ -127,7 +127,7 @@ def run_experiment(self, qobj_experiment: QasmQobjExperiment, **options):
 
         sim = ddsim.HybridCircuitSimulator(qobj_experiment, seed, hybrid_mode, nthreads)
 
-        shots = options['shots']
+        shots = options.get('shots', 1024)
```

Comment:
IBM changed the default number of shots on their devices from 1024 to 4000 quite recently. Do we want to take this opportunity and change it here as well? The same, of course, goes for the other simulators.